src/test/GradleTestDevModeActions.ts

```diff
@@ -263,6 +264,10 @@ it('View test report for gradle project', async () => {
 }).timeout(30000);
 
 
+  // Now, clear the command history of the "command palette" to avoid receiving "recently used" suggestions. This action should be performed at the end of Gradle Project tests.
```

Comment:
what happens if we don't clear the history? We should explain that in the comment here.